includes/class-wc-payments.php

```diff
@@ -667,6 +667,12 @@ public static function init_rest_api() {
 			$survey_controller = new WC_REST_Payments_Survey_Controller( self::get_wc_payments_http() );
 			$survey_controller->register_routes();
 		}
+
+		if ( WC_Payments_Features::is_documents_section_enabled() ) {
+			include_once WCPAY_ABSPATH . 'includes/admin/class-wc-rest-payments-documents-controller.php';
+			$documents_controller = new WC_REST_Payments_Documents_Controller( self::$api_client );
+			$documents_controller->register_routes();
+		}
```

Comment:
💯 - thanks for making sure the new API endpoints are not added if the feature flag is not enabled!

Reply:
No problem 😁